pypom/selenium_driver.py

```diff
@@ -0,0 +1,133 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from zope.interface import (
+    implementer,
+    Interface,
+)
+
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver import (
+    Remote,
+    Opera,
```

Comment:
Remote would cover a connection to a Selenium server, which could be running Firefox, Chrome, etc. It wouldn't cover a direct instantiation of the Firefox driver. I suspect we'd need to include all of the driver classes from https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/__init__.py